theano/tensor/nnet/abstract_conv.py

```diff
@@ -1849,14 +1992,16 @@ def __init__(self,
                  subsample=(1, 1),
                  filter_flip=True,
                  filter_dilation=(1, 1),
-                 num_groups=1):
+                 num_groups=1,
+                 unshared=False):
```

Comment:
It seem it is supported to combine num_groups and unshared. Make that clear in the doc.